### ISSUE TITLE
Web Annotations Display

### ISSUE THREAD

**Issue by cyork** [jhu-digital-manuscripts/AnIOp#34]
- [x] Use Mirador 3 to display web annotations generated and defined by this project with associated IIIF resources

- [x] Using diagramming software, generate mockups of what web annotations would look like in Mirador 3

**Comment by GroovinChip** [comment]
As mentioned in #30 and #33, the Mirador sidebar will support Web Annotations in the near future. This will play into design ideas and decisions because we can utilize that area for some of our data.

**Comment by GroovinChip** [comment]
Jack has opened a pull request for this code: https://github.com/ProjectMirador/mirador/pull/2898

**Comment by GroovinChip** [comment]
I have enabled a much nicer way of viewing web annotations:
![firefox_5MQh4N3Llw](https://user-images.githubusercontent.com/4250470/71028280-feeb1500-20da-11ea-8dbf-695886860089.png)
I obtained a crucial piece of information that made this possible - that Mirador returns a MenuItem component from the MaterialUI library it uses. That's why my prior implementation looked so weird! The correct approach is to give the MenuItem a Button and open a Dialog from there.
